<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="APL-5CD010D4Y6" ignored="0" name="Test" tests="2" failures="2" timestamp="2020-08-07T19:24:49 IST" time="24.682" errors="0">
  <testcase name="@AfterMethod afterMethod" time="0.001" classname="TestScripts.BaseSuiteFile">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at TestScripts.BaseSuiteFile.afterMethod(BaseSuiteFile.java:282)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- @AfterMethod afterMethod -->
  <testcase name="@AfterMethod afterMethod" time="-1.596808489171E9" classname="TestScripts.BaseSuiteFile">
    <skipped/>
  </testcase> <!-- @AfterMethod afterMethod -->
  <testcase name="@AfterClass afterClass" time="-1.596808489172E9" classname="TestScripts.BaseSuiteFile">
    <skipped/>
  </testcase> <!-- @AfterClass afterClass -->
  <testcase name="@AfterTest afterTest" time="-1.596808489173E9" classname="TestScripts.BaseSuiteFile">
    <skipped/>
  </testcase> <!-- @AfterTest afterTest -->
  <testcase name="Caseid_02_05_AlertsPage_Raja" time="21.273" classname="TestScripts.BaseSuiteFile">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at LibraryFiles.ReUsableLibrary.launchApplication(ReUsableLibrary.java:278)
at TestScripts.Caseid_02_05_AlertsPage_Raja.Caseid_02_05_AlertsPage(Caseid_02_05_AlertsPage_Raja.java:21)
at TestScripts.BaseSuiteFile.Caseid_02_05_AlertsPage_Raja(BaseSuiteFile.java:241)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- Caseid_02_05_AlertsPage_Raja -->
  <testcase name="Case_04_Frames_Raja" time="24.276" classname="TestScripts.BaseSuiteFile">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at LibraryFiles.ReUsableLibrary.launchApplication(ReUsableLibrary.java:278)
at TestScripts.Case_04_Frames_Raja.Case_04_Frames(Case_04_Frames_Raja.java:23)
at TestScripts.BaseSuiteFile.Case_04_Frames_Raja(BaseSuiteFile.java:250)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- Case_04_Frames_Raja -->
</testsuite> <!-- Test -->
